### PR TITLE
Update clan.js

### PR DESCRIPTION
**Pull request by Safe870**
Few more translations!

### CODE REVIEW

File: loc/ui/clan.js [hunk 1]
```diff
@@ -6,7 +6,17 @@ export default {
     fr: 'Gestionnaire du clan',
     hi: 'दल प्रबंधक',
     es: 'Gerente de clan',
-    de: 'Clan-Manager'
+    de: 'Clan-Manager',
+    bg: 'Мениджър на клана',
+    cs: 'Clan Manager',
```
Comment (zbot473) [inline loc/ui/clan.js]: Are you sure about these? it seems too easy...

Comment (KaiserCheezel) [reply loc/ui/clan.js]: The dash could be messing the translator up. Guessing it was done with Google Translate in that case

Comment (zbot473) [reply loc/ui/clan.js]: Possibly.

Comment (Safe870) [reply loc/ui/clan.js]: No, I didn't use Google Translate. Let me check again.

Comment (Safe870) [reply loc/ui/clan.js]: мениджър на клана=Clan Manager in bg.
manažer klanu=Clan Manager in cs.
Sorry I must've not focused.

Comment (zbot473) [reply loc/ui/clan.js]: Ok fair enough

**Pull request merged**